contracts/src/Initializer.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+pragma solidity 0.8.25;
+
+import {MerkleProof} from "openzeppelin/utils/cryptography/MerkleProof.sol";
+import {Verification} from "./Verification.sol";
+
+import {AgentExecutor} from "./AgentExecutor.sol";
+import {Agent} from "./Agent.sol";
+import {
+    OperatingMode,
+    ParaID,
+    TokenInfo,
+    MultiAddress,
+    Channel,
+    ChannelID
+} from "./Types.sol";
+import {Upgrade} from "./Upgrade.sol";
+import {IInitializable} from "./interfaces/IInitializable.sol";
+import {IUpgradable} from "./interfaces/IUpgradable.sol";
+import {ERC1967} from "./utils/ERC1967.sol";
+import {Address} from "./utils/Address.sol";
+import {SafeNativeTransfer} from "./utils/SafeTransfer.sol";
+import {Call} from "./utils/Call.sol";
+import {Math} from "./utils/Math.sol";
+import {ScaleCodec} from "./utils/ScaleCodec.sol";
+
+import {CoreStorage} from "./storage/CoreStorage.sol";
+import {PricingStorage} from "./storage/PricingStorage.sol";
+import {AssetsStorage} from "./storage/AssetsStorage.sol";
+import {OperatorStorage} from "./storage/OperatorStorage.sol";
+
+import {Constants} from "./Constants.sol";
+
+import {UD60x18, ud60x18, convert} from "prb/math/src/UD60x18.sol";
+
+library Initializer {
+    error Unauthorized();
+
+    // Initial configuration for bridge
+    struct Config {
+        OperatingMode mode;
+        /// @dev The fee charged to users for submitting outbound messages (DOT)
+        uint128 deliveryCost;
+        /// @dev The ETH/DOT exchange rate
+        UD60x18 exchangeRate;
+        /// @dev The extra fee charged for registering tokens (DOT)
+        uint128 assetHubCreateAssetFee;
+        /// @dev The extra fee charged for sending tokens (DOT)
+        uint128 assetHubReserveTransferFee;
+        /// @dev extra fee to discourage spamming
+        uint256 registerTokenFee;
+        /// @dev Fee multiplier
+        UD60x18 multiplier;
+        /// @dev Optional rescueOperator
+        address rescueOperator;
+        uint8 foreignTokenDecimals;
+        uint128 maxDestinationFee;
+        address weth;
```

Comment:
Move weth to the first member.
```suggestion
        /// @dev The WETH address used for wrapping ETH
        address weth;
        uint8 foreignTokenDecimals;
        uint128 maxDestinationFee;
```
Commit:
https://github.com/Snowfork/snowbridge/pull/1343/commits/e1a85782ec1e58d8fa1f8bbdfc6cd4c6b247d470